server/db/models/product.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  title: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  description: {
+    type: Sequelize.TEXT,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    },
+    instructions: {
+      type: Sequelize.TEXT,
+      allowNull: false,
+      validate: {
+        notEmpty: true
+      }
+    }
+  },
+  imageUrl: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  price: {
+    type: Sequelize.INTEGER,
```

Comment:
if this type doesnt support numbers with decimals - can you make sure that it does?